src/cryptonote_protocol/cryptonote_protocol_handler.inl

```diff
@@ -1168,6 +1168,8 @@ skip:
             if (ELPP->vRegistry()->allowed(el::Level::Debug, "sync-info"))
               timing_message += std::string(": ") + m_block_queue.get_overview();
             MGINFO_YELLOW(context << " Synced " << m_core.get_current_blockchain_height() << "/" << m_core.get_target_blockchain_height()
+                << " " << (m_core.get_current_blockchain_height() / m_core.get_target_blockchain_height() * 100) << "% ("
```

Comment:
This should check get_target_blockchain_height is not zero. I'm not convinced it can't happen here, as it gets updated when a node disconnects. Also probably should read 99.9% or something if it's not yet the, similar to the status command.